### PR TITLE
Update Cycle Tests Examples

### PR DESCRIPTION
**Pull request by davideziviani**
Updated the examples to reflect the changes within ACHP

### CODE REVIEW

File: CycleTests/ACTest.py [hunk 1]
```diff
@@ -1,19 +1,28 @@
...
 Cycle.ImposedVariable = 'Subcooling'
+Cycle.CycleType = 'DX'
 Cycle.DT_sc_target = 7.0
 Cycle.Mode='AC'
 Cycle.Ref='R410A'
 Cycle.Backend='TTSE&HEOS' #Backend for refrigerant properties calculation: 'HEOS','TTSE&HEOS','BICUBIC&HEOS','REFPROP','SRK','PR'
+Cycle.Oil = 'POE32'
+Cycle.shell_pressure = 'low-pressure'
+Cycle.EvapSolver = 'Moving-Boundary' #choose the type of Evaporator solver scheme ('Moving-Boundary' or 'Finite-Element')
+Cycle.EvapType = 'Fin-tube' #if EvapSolver = 'Moving-Boundary', choose the type of evaporator ('Fin-tube' or 'Micro-channel')
```
Comment (abahman) [inline CycleTests/ACTest.py]: Micro-channel evaporator is not implemenetd here. I suggest we update the comment

File: CycleTests/PHECycle.py [hunk 2]
```diff
@@ -212,7 +246,16 @@ def OBJECTIVE(x):
         Cycle.DT_sc_target = 7.0
         Cycle.Charge_target = 3.3
         Cycle.Ref='R410A'
+        Cycle.Backend='TTSE&HEOS' #Backend for refrigerant properties calculation: 'HEOS','TTSE&HEOS','BICUBIC&HEOS','REFPROP','SRK','PR'
+        Cycle.Oil = 'POE32'
+        Cycle.shell_pressure = 'low-pressure'
         Cycle.SecLoopFluid = 'Water'
+        Cycle.Backend_SLF = 'INCOMP' #backend of SecLoopFluid
+        Cycle.EvapSolver = 'Moving-Boundary' #choose the type of Evaporator solver scheme ('Moving-Boundary' or 'Finite-Element')
+        Cycle.EvapType = 'Fin-tube' #if EvapSolver = 'Moving-Boundary', choose the type of evaporator ('Fin-tube' or 'Micro-channel')
```
Comment (abahman) [inline CycleTests/PHECycle.py]: Update comment here as well

File: CycleTests/SampleCycles.py [hunk 3]
```diff
@@ -395,9 +424,16 @@ def SampleDXACSystem(Calculate=True):
     Cycle.Charge_target = 2.8
     Cycle.Mode='AC' 
     Cycle.Ref='R410A'
+    Cycle.Oil = 'POE32'
+    Cycle.shell_pressure = 'low-pressure'
     Cycle.TestName='DXAC-0014'  #this and the two next lines can be used to specify exact test conditions
     Cycle.TestDescription='shows application of DXAC system'
     Cycle.TestDetails='This is the sample cycle for the DXAC system which can be modified for other systems'
+    Cycle.EvapSolver = 'Moving-Boundary' #choose the type of Evaporator solver scheme ('Moving-Boundary' or 'Finite-Element')
+    Cycle.EvapType = 'Fin-tube' #if EvapSolver = 'Moving-Boundary', choose the type of evaporator ('Fin-tube' or 'Micro-channel')
```
Comment (abahman) [inline CycleTests/SampleCycles.py]: update comment here!

File: CycleTests/ACTest.py [hunk 4]
```diff
@@ -1,19 +1,28 @@
 '''This code is for Direct Expansion in Cooling Mode'''
```
Comment (ibell) [inline CycleTests/ACTest.py]: You also need a from __future__ import print_function for python 2/3 support

File: CycleTests/ACTest.py [hunk 5]
```diff
@@ -92,10 +103,16 @@
...
 
 Cycle.Evaporator.FinsType = 'WavyLouveredFins'        #WavyLouveredFins, HerringboneFins, PlainFins
-Cycle.Evaporator.Ref=Cycle.Ref
-Cycle.Evaporator.Backend=Cycle.Backend              #Backend for refrigerant properties calculation
 Cycle.Evaporator.Verbosity=0
-Cycle.Evaporator.DT_sh=5
+Cycle.Evaporator.DT_sh=5                    #target superheat
+
+# ----------------------------------
+#       Expanison device Parameters
```
Comment (ibell) [inline CycleTests/ACTest.py]: Typo and capitalization inconsistency

File: CycleTests/HPTest.py [hunk 6]
```diff
@@ -1,7 +1,7 @@
 '''This code is for Direct Expansion in Heating Mode'''
```
Comment (ibell) [inline CycleTests/HPTest.py]: You also need a from __future__ import print_function for python 2/3 support

File: CycleTests/SLACTest.py [hunk 7]
```diff
@@ -1,6 +1,6 @@
 '''This code is for secondary loop cycle in AC Mode'''
```
Comment (ibell) [inline CycleTests/SLACTest.py]: You also need a from __future__ import print_function for python 2/3 support

**Pull request merged**